### PR TITLE
Merge master HEAD into openj9-staging

### PR DESCRIPTION
**Pull request by JasonFengJ9**
Resolved conflicts:
```
	src/java.base/share/classes/java/lang/Thread.java
```

Signed-off-by: Jason Feng <fengj@ca.ibm.com>

### CODE REVIEW

File: src/java.base/share/classes/java/lang/Thread.java [hunk 1]
```diff
@@ -1685,26 +1645,7 @@ void exit() {
...
-            throw new UnsupportedOperationException();
-
-        // A zero status value corresponds to "NEW", it can't change to
-        // not-NEW because we hold the lock.
-        if (started && (eetop != NO_REF)) {
-            resume(); // Wake up thread if it was suspended; no-op otherwise
-        }
-
-        // The VM can handle all thread states
-        stop0(new ThreadDeath());
```
Comment (keithc-ca) [inline src/java.base/share/classes/java/lang/Thread.java:1707]: This is the time to remove `stop0()` and `stopImpl()`.

**Review by keithc-ca — CHANGES_REQUESTED** [review]

File: src/java.base/share/classes/java/lang/Thread.java (hunk 1 above)
Comment (JasonFengJ9) [reply src/java.base/share/classes/java/lang/Thread.java:1707]: Removed `stop0()`, `stopImpl()` and `stopCalled`.
Created OpenJ9 PR https://github.com/eclipse-openj9/openj9/pull/15963

Comment (keithc-ca) [reply src/java.base/share/classes/java/lang/Thread.java:1707]: Thanks, I had missed that `stopCalled` could be removed.

**Comment by keithc-ca** [comment]
jenkins compile amac jdknext depends eclipse-openj9/openj9#15963

**Comment by keithc-ca** [comment]
jenkins compile amac jdknext depends eclipse-openj9/openj9#15963

**Pull request merged**